### ISSUE TITLE
Upgrade logback to the latest build of 1.3.0

### ISSUE THREAD

**Issue by robotdan** [FusionAuth/fusionauth-issues#1530]
## Upgrade logback 

The logging framework used by FusionAuth is logback. While this library does not have the same vulnerability found in Log4J, due to this post on the logback website, we are updating to the latest version out of an abundance of caution. 

### Description

From: http://logback.qos.ch/news.html
```

You can receive logback-related announcements by subscribing to the QOS.ch announce mailing list.

16th of December, 2021, Release of version 1.2.9

We note that the vulnerability mentioned in CVE-2021-42550 requires write access to logback's configuration file as a prerequisite. Please understand that log4Shell and CVE-2021-42550 are of different severity levels.
In response to CVE-2021-42550 (aka LOGBACK-1591) we have decided to make the following steps.

1) Hardened logback's JNDI lookup mechanism to only honor requests in the java: namespace. All other types of requests are ignored. Many thanks to Michael Osipov for suggesting this change and providing the relvant PR.

2) SMTPAppender was hardened.

3) Temporarily removed DB support for security reasons.

4) Removed Groovy configuration support. As logging is so pevasive and configuration with Groovy is probably too powerful, this feature is unlikely to be reinstated for security reasons.

We note that the aforementioned vulnerability requires write access to logback's configuration file as a prerequisite. Please understand that log4Shell/CVE-2021-44228 and CVE-2021-42550 are of different severity levels. A successul RCE attack with CVE-2021-42550 requires all of the following conditions to be met:

write access to logback.xml
use of versions < 1.2.9
reloading of poisoned configuration data, which implies application restart or scan="true" set prior to attack
As an additional extra precaution, in addition to upgrading to logback version 1.2.9, we also recommend users to set their logback configuration files as read-only.

16th of December, 2021, Release of version 1.3.0-alpha11

Note that 1.3.0-alpha11 contains the same security related changes as 1.2.9.

The 1.3.x series is JPMS/Jigsaw/Java 9 modularized and requires slf4j-api version 2.0.x. However, the 1.3.x series requires Java 8 or later at runtime.
The 1.3.x series is Jigsaw/Java 9 modularized and requires slf4j-api version 2.0.x. Moreover, the 1.3.x series requires Java 8 or later at runtime whereas building logback from source requires Java 9. 

Joran, logback's configuration system, has been rewritten to use an internal representation model which can be processed separately. As a side-effect, logback configuration scripts are now largely order-free. For example, appenders can now be defined after they are first referenced in a logger. Moreover, unreferenced appenders are no longer instantiated. Given the breadth of the changes in Joran codebase, support for SiftingAppender and Groovy configuration have been dropped temporarily.

• Migrated from javax.servlet to jakarta.servlet. This entails migration to Tomcat version 10.0.10 and Jetty version 11.0.6 in logback-access. This fixes LOGBACK-1575 reported by Daniel Svensson. Note that Jetty version 11 requires Java version 11 or later.
```

### Additional context
- http://logback.qos.ch/news.html

Is this a question about how to use FusionAuth? Please consider posting on the [FusionAuth forum](https://fusionauth.io/community/forum/) instead.

### Community guidelines
All issues filed in this repository must abide by the [FusionAuth community guidelines](https://fusionauth.io/community/forum/topic/1000/code-of-conduct).

**Comment by robotdan** [comment]
Rolling this back for now, logback has a bug in their latest version that causes dates to be printed as epoch.

Once they fix this bug, we can take another look. 

- https://jira.qos.ch/browse/LOGBACK-1596
- https://jira.qos.ch/browse/LOGBACK-1599

**Comment by mooreds** [comment]
Can we go to 1.2.9 instead of 1.3.0?

**Comment by robotdan** [comment]
Unfortunately can't use `1.2.9`, we have other deps required for Java 17 that require 1.3.0. The above Jira issue is resolved, and a patch is out in `1.3.0-alpha12`.
